src/components/include/policy/policy_external/policy/policy_manager.h

```diff
@@ -85,8 +88,10 @@ class PolicyManager : public usage_statistics::StatisticsManager,
    * @param pt_content PTU as binary string
    * @return true if successfully
```

Comment:
as i undrestand, this method is no longer returning `bool`

Reply:
@mked-luxoft good point. Description was updated in https://github.com/smartdevicelink/sdl_core/commit/d6e668e24d4444f10daf55c953b21af69c1be690